example/prepare_binaries.sh

```diff
@@ -0,0 +1,34 @@
+export NBS_ROOT=".."
+YDBD_BIN="$NBS_ROOT/contrib/ydb/apps/ydbd/ydbd"
+NBSD_BIN="$NBS_ROOT/cloud/blockstore/apps/server/nbsd"
+BLOCKSTORE_CLIENT_BIN="$NBS_ROOT/cloud/blockstore/apps/client/blockstore-client"
+DISK_AGENT_BIN="$NBS_ROOT/cloud/blockstore/apps/disk_agent/diskagentd"
+BLOCKSTORE_NBD_BIN="$NBS_ROOT/cloud/blockstore/tools/nbd/blockstore-nbd"
+
+for bin in $YDBD_BIN $NBSD_BIN $BLOCKSTORE_CLIENT_BIN $DISK_AGENT_BIN $BLOCKSTORE_NBD_BIN
+do
+  if ! test -f $bin; then
+    echo "$bin not found, build all targets first"
+    return 1
+  fi
+done
+
+function ydbd () {
```

Comment:
the correct syntax for bash function is either
```bash
function myfunc {
}
```
or
```bash
myfunc () {
}
```
no need to have both `function` and `()`

Reply:
According to [manual](https://www.gnu.org/software/bash/manual/bash.html#Shell-Functions): 
> This defines a shell function named fname. The reserved word function is optional. If the function reserved word is supplied, the parentheses are optional. 

So using both `function` word and parentheses is also correct syntax